plugins/source/aws/resources/services/lambda/functions.go

```diff
@@ -62,6 +63,24 @@ func Functions() *schema.Table {
 				Type:     arrow.BinaryTypes.String,
 				Resolver: resolveRuntimeManagementConfig,
 			},
+			{
+				Name: "code",
```

Comment:
What's the reason we need to declare `code` and `configuration`?

Reply:
This PR has all of the functional changes but fails because of `gen` changes
https://github.com/cloudquery/cloudquery/actions/runs/6433569162/job/17470896579